arrow-libs/core/arrow-core/src/main/kotlin/arrow/core/extensions/either/apply/EitherApply.kt

```diff
@@ -109,7 +124,10 @@ fun <L, A, B, Z> mapN(
   "EXTENSION_SHADOWED_BY_MEMBER",
   "UNUSED_PARAMETER"
 )
-@Deprecated("@extension kinded projected functions are deprecated", ReplaceWith("arg0.zip(arg1, arg2) { a, b, c -> arg3(Tuple3(a, b, c)) }", "arrow.core.zip", "arrow.core.Tuple3"))
+@Deprecated(
+  "@extension kinded projected functions are deprecated",
+  ReplaceWith("arg0.zip(arg1, arg2) { a, b, c -> arg3(Tuple3(a, b, c)) }", "arrow.core.zip", "arrow.core.Tuple3")
```

Comment:
Should these be removing the tupling since the zip functions don't take their args tupled?
```suggestion
  ReplaceWith("arg0.zip(arg1, arg2) { a, b, c -> arg3(a, b, c) }", "arrow.core.zip")
```

Reply:
The reason for the tupling here is since the `arg3` lambda is of type `f: (Tuple3<A, B, C> -> Z)`, so we need to create the tuple to pass it to the lambda.